source/game/kart/KartObjectProxy.cc

```diff
@@ -317,6 +322,22 @@ bool KartObjectProxy::hasFloorCollision(const WheelPhysics *wheelPhysics) const
     return wheelPhysics->hitboxGroup()->collisionData().bFloor;
 }
 
+/// @addr{0x8058539C}
+std::pair<EGG::Vector3f, EGG::Vector3f> KartObjectProxy::cannonPosRot() {
```

Comment:
```suggestion
std::pair<EGG::Vector3f, EGG::Vector3f> KartObjectProxy::getCannonPosRot() {
```
Because this derives the position and rotation rather than outright getting it, we prefer `get` here.